application.rb

```diff
@@ -0,0 +1,184 @@
+require_relative 'services/login_service.rb'
+require_relative 'services/sign_up_service.rb'
+require_relative 'services/task_manager.rb'
+
+class Application
+  @@TaskManager = nil 
+
+  def self.boot
+    @@TaskManager = TaskManager.new
+    start_event_loop 
+  end
+
+  def self.assign_task(user)
+    puts 'enter assignee user id'
+    assignee_user_id = gets.chomp
+    puts 'Enter Description about task'
+    description = gets.chomp
+    puts 'Enter Due_date Format (YYYY/MM/DD)'
+    due_date = gets.chomp
+    puts 'Enter priority eg: 1, 2, 3'
+    priority = gets.chomp
+    @@TaskManager.add_task(
+      assignee_user_id: assignee_user_id,
+      description: description,
+      due_date: due_date,
+      priority: priority, 
+      user: user
+    )
+  end
+
+  def self.delete_task
+    puts "Enter Task id you want to delete"
+    delete_id = gets.chomp.to_i
+    @@TaskManager.delete_task(delete_id)
+  end
+
+  def self.for_admin(user)
+    system('clear')
+    puts "Welcome to Admin Panel"
+    puts "---------------------------"
+    loop do 
+      puts "\nMenu:"
+      puts "1. Add Task"
+      puts "2. Get All tasks"
+      puts "3. Delete task"
+      puts "4. Exit"
+      choice = gets.chomp.to_i
+  
+      case choice
+      when 1 
+        assign_task(user)
+      when 2
+        @@TaskManager.show_all_tasks
+      when 3
+        delete_task
+      when 4
+        puts("Exiting.......")
+        exit
+      else
+        puts("Invalid Choice. Please Enter a valid choice")    
+      end
+    end
+  end
+
+  def self.change_status
+    puts "Enter Task id whose status want to change"
+    task_id = gets.chomp.to_i
+    puts "Available Statuses: pending, due, in progress, under_review, closed"
+    puts "Enter the new status:"
+    new_status = gets.chomp
+    @@TaskManager.change_the_status(task_id, new_status)
```

Comment:
`update_status`. English articles like a,an, the can be skipped in naming anything (methods/variables/classes)